projects/ngx-prx-styleguide/src/lib/datepicker/datepicker.component.ts

```diff
@@ -3,14 +3,9 @@ import { Component, Input, Output, EventEmitter, AfterViewInit, ViewChild,
 
 import * as Pikaday from 'pikaday';
 
-import * as momentNs from 'moment';
-
-// Moment doesn't export the type for the moment() function itself, so I need to set one up
```

Comment:
This is no longer needed as we've added third-party types that are more complete (`@types/moment-timezone`)